web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/Body.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mobilitydata.gtfsvalidator.web.service;
+
+// Body.Message is the payload of a Pub/Sub event. Please refer to the docs for
+// additional information regarding Pub/Sub events.
+public class Body {
```

Comment:
Does this class have to be called body?  Could it be given a more descriptive name or would that break something?